library/Data/NBA/Stats.hs

```diff
@@ -19,6 +19,7 @@ module Data.NBA.Stats (
     -- * Simple API
     getSplitRows,
     getSplitRow,
+    splitRows,
```

Comment:
I think this diff should also include `splitRow`, `splitRowsGeneric`, and `splitRowGeneric`.